crates/uv-resolver/src/pubgrub/package.rs

```diff
@@ -166,6 +166,26 @@ impl PubGrubPackage {
         }
     }
 
+    /// Returns the extra name associated with this PubGrub package, if it has
+    /// one.
+    #[allow(dead_code)]
```

Comment:
why the `#[allow(dead_code)]`?

Reply:
I think I added this before I realized I wanted to do a refactor. It isn't used yet, but I believe I'll need it for the conflicting extras work. I'll remove this for now.